pkgs/tools/misc/passmark-performancetest/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, curl
+, unzip
+, ncurses5
+, autoPatchelfHook
+}:
+let
+  version = "10.2.1003";
+  versionUri = builtins.replaceStrings ["."] ["_"] version;
+
+  sources = {
+    "x86_64-linux" = {
+      url = "https://www.passmark.com/downloads/pt_linux_x64_${versionUri}.zip";
+      hash = "sha256-H9IusuDkyFRD4LVcIF5eGVsIeYgzCkCNJR+eZagqQOc=";
+    };
+    "aarch64-linux" = {
+      url = "https://www.passmark.com/downloads/pt_linux_arm64_${versionUri}.zip";
+      hash = "sha256-v7qxVR2eM5w4178J3+sI2Tpi5SEteQQbabXgin6RUUk=";
+    };
+  };
+in
+stdenv.mkDerivation rec {
+  inherit version;
+  pname = "passmark-performancetest";
+
+  src = fetchurl (sources.${stdenv.system});
```

Comment:
This is missing an or otherwise eval on other platforms fails with a syntax error.

Reply:
> I saw eval will only take place based on meta.platform

eval is for all platforms everything, meta.platform just limits what is being build afterwards.

> I use the same style as Geekbench

then it is doing the same mistake.

---

see https://github.com/NixOS/nixpkgs/blob/cc8f80e61865526340b36e7f092d00189649836c/nixos/modules/virtualisation/qemu-vm.nix#L912

Reply:
Thanks, fixed!